ironfish/src/rpc/adapters/socketAdapter/socketAdapter.ts

```diff
@@ -193,6 +197,21 @@ export abstract class RpcSocketAdapter implements IRpcAdapter {
 
       const message = result.result.data
 
+      if (this.router == null || this.router.server == null) {
+        this.emitResponse(client, this.constructUnmountedAdapter())
+        return
+      }
```

Comment:
This has to go back where it was to properly respond to the message and handle it properly.